open-company/open-company-web
# Headlines

card: https://trello.com/c/rvuKdS9G

to test:
- open a company
- check the headlines
- update the headlines of some sections
- reload and check that the headlines were saved
- check the headline char counter
- check the placeholder with no text
- check CI
- merge
